infrastructure/main.tf

```diff
@@ -147,37 +147,60 @@ data "vault_generic_secret" "barApi" {
+  microservice_key_names = {
+    "CCD_ADMIN"                   = "microservicekey-ccd-admin"
+    "CCD_DATA"                    = "microservicekey-ccd-data"
+    "CCD_DEFINITION"              = "microservicekey-ccd-definition"
+    "CCD_GW"                      = "microservicekey-ccd-gw"
+    "CCD_PS"                      = "microservicekey-ccd-ps"
+    "CMC"                         = "microservicekey-cmc"
+    "CMC_LEGAL_FRONTEND"          = "microservicekey-cmcLegalFrontend"
+    "CMC_CLAIM_STORE"             = "microservicekey-cmcClaimStore"
+    "DIVORCE"                     = "microservicekey-divorce"
+    "DIVORCE_FRONTEND"            = "microservicekey-divorce-frontend"
+    "DIVORCE_CCD_SUBMISSION"      = "microservicekey-divorceCcdSubmission"
+    "DIVORCE_CCD_VALIDATION"      = "microservicekey-divorceCcdValidation"
+    "DIVORCE_DOCUMENT_UPLOAD"     = "microservicekey-divorceDocumentUpload"
+    "DIVORCE_DOCUMENT_GENERATOR"  = "microservicekey-divorceDocumentGenerator"
+    "DRAFT_STORE_TESTS"           = "microservicekey-draftStoreTests"
+    "JOBSCHEDULER"                = "microservicekey-platformJobScheduler"
+    "REFERENCE"                   = "microservicekey-reference"
+    "SSCS"                        = "microservicekey-sscs"
+    "SSCS_TRIBUNALS_CASE"         = "microservicekey-sscs-tribunals-case"
+    "PROBATE_FRONTEND"            = "microservicekey-probate-frontend"
+    "PROBATE_BACKEND"             = "microservicekey-probate-backend"
+    "SEND_LETTER_CONSUMER"        = "microservicekey-send-letter-consumer"
+    "SEND_LETTER_TESTS"           = "microservicekey-send-letter-tests"
+    "EM_GW"                       = "microservicekey-em-gw"
+    "FINREM"                      = "microservicekey-finrem"
+    "FINREM_DRAFT_STORE"          = "microservicekey-finrem-draft-store"
+    "JUI_WEBAPP"                  = "microservicekey-jui-webapp"
+    "PUI_WEBAPP"                  = "microservicekey-pui-webapp"
+    "COH_COR"                     = "microservicekey-coh-cor"
+    "BULK_SCAN_PROCESSOR"         = "microservicekey-bulk-scan-processor"
+    "BULK_SCAN_PROCESSOR_TESTS"   = "microservicekey-bulk-scan-processor-tests"
+    "BAR_API"                     = "microservicekey-bar-api"
+  }
 
-module "s2s-api" {
-  source       = "git@github.com:hmcts/moj-module-webapp.git?ref=master"
-  product      = "${var.product}-${var.component}"
-  location     = "${var.location}"
-  env          = "${var.env}"
-  ilbIp        = "${var.ilbIp}"
-  subscription = "${var.subscription}"
-  capacity     = "${var.capacity}"
-  common_tags  = "${var.common_tags}"
+  microservice_secret_names = "${values(local.microservice_key_names)}"
+
+  microservice_key_settings = "${zipmap(
+                                    formatlist("MICROSERVICEKEY_%s", keys(local.microservice_key_names)),
+                                    data.azurerm_key_vault_secret.microservice_keys.*.value
+                                )}"
+
+  core_app_settings = {
```

Comment:
why are these still here?
looks like they are just getting merged on top of the new ones?

Reply:
I'm planning to leave them just for now (to be removed as part of current story), so that it's easy to verify that it's the same secrets (they have `MICROSERVICEKEY_` vs `MICROSERVICEKEYS_` prefix). All the hashicorp vault stuff will be removed, as well.